src/stars.css

```diff
@@ -0,0 +1,53 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+
+/* #region stars */
```

Comment:
remove all the comments, you already separated blocks info files, so there is no need it comments